### ISSUE TITLE
Color variable names 

### ISSUE THREAD

**Issue by jamieg270** [imperavi/superkube#4]
Not sure this is an issue or by design. In previous versions the root color variables display with a name that mapped to the Pallete/Color map. This made it super easy to inspect the code of a site and see what color was mapped from where and applied where.

In version 2 these show as --c34, --c35 etc.. There is no descriptor. It is much more difficult to inspect the site and identify which color variable maps to which color on the pallete/colors map.

**Comment by imperavi** [comment]
Indeed, variables like --c34 are now misleading. We did this to reduce the size of the css. Because a large number of color variables can take 15-20kb or more. That is not good. 

But now we have found a new solution, which on the one hand will not greatly increase the size of the generated css, but on the other hand will make the names of color variables logical. And in general it will make color specifying more logical. 

In the previous version, the color variables were semantic, like: 

--c-heading-default
--c-button-default-base-text

In the future (probably in the next update) we want to make color variables related to the palette, i.e. 

--c-black
--c-primary-base
etc
